tests/acceptance/Makefile

```diff
@@ -30,3 +30,6 @@ clean:
 	rm -f env-tester.bin
 	rm -f client.bin
 
+#.PHONY: path-to-referenced-charts
```

Comment:
commented code, Leo why?

Reply:
because there are some components which has incorrect keys in `values.yaml`, until it is corrected Makefile target has to be commented. For more details click [here](https://github.com/kyma-project/test-infra/issues/588#issuecomment-470487256)